tests/debugging/exception/test_auto_instrument.py

```diff
@@ -24,8 +25,10 @@ def setUp(self):
         super(ExceptionDebuggingTestCase, self).setUp()
         self.backup_tracer = ddtrace.tracer
         ddtrace.tracer = self.tracer
+        _third_party_packages().remove("ddtrace")
```

Comment:
Seems weird. How this related to the infinite loop bug?

Or this something else to make CI pass?

Reply:
This is needed because `ddtrace` is part of the third-party library list. We need to remove it for the test to make it detect our sources as "user code".